README.md

```diff
@@ -1,2 +1,156 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+-**HTML & CSS**
+
+-**JAVASCRIPT**
+
+-**WEBPACK**
+
+-**JEST**
+
+### Key Features <a name="key-features"></a>
+
+- **TV Shows Listing: Browse a collection of popular TV shows with their details, such as title, description, and airing date.**
+
+- **Like Shows: Users can like their favorite TV shows, and the app will keep track of their preferences.**
+
+- **Comments: Users can leave comments on TV shows to share their opinions and discuss with other users.**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Visit our live project]()
+
+## 🚀 Description Video<a name="video-demo"></a>
+
+- [Link to our description video](https://drive.google.com/file/d/17jh4JKd81T7QvOJDLy_CoZUjgQS_UlN_/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+sudo apt install git-all
+or
+npm install
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone  https://github.com/Ayokunnumi1/Mini-Netflix-Websit.git
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ayoknnumi**
+
+- GitHub: [@Ayokunnumi1][https://github.com/Ayokunnumi1]
+- Twitter: [@Ayokunnumi1] [https://twitter.com/AyokunnumiA]
+- LinkedIn: [Ayokunnumi] [https://www.linkedin.com/in/ayokunnumiomololu]
+
+👤 **Hydaline**
+
+- GitHub: [https://github.com/hydaline-code]
+- Twitter: [@hydalineC][https://twitter.com/@hydalineC]
+- LinkedIn: [https://www.linkedin.com/in/charlene-djoungang-0851aa21a/]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Add a About page ]**
+
+- **[Add more interactivity ]**
+
+- **[Add a contact form page ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/https://github.com/Ayokunnumi1).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project and follow me !
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to give thanks to Microverse for creating this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/https://github.com/Ayokunnumi1/dev/LICENSE) licensed.
```

Comment:
- [ ] [OPTIONAL] Kindly replace the MIT License link with a working link as this particular link is currently broken. The License is important to declare to other developers that you own this project and it is a hard requirement.

Reply:
Alright, i have worked on this as well, thank you